.env

```diff
@@ -0,0 +1,17 @@
+# Environment variables declared in this file are automatically made available to Prisma.
```

Comment:
.env file should me committed in git. Keep it in local.

Reply:
it's already committed. Maybe came with master branch